docs/advanced_topics/storage_rewrites.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: default
-nav_order: 4
+nav_order: 40
 parent: Advanced topics
-title: Configuration & storage rewrite
+title: User Configuration - Migration & Duplication
```

Comment:
User Configuration - Migration & Duplication -> User configuration - migration & duplication

I think it would fit better here not to capitalize the individual words, at least that's how we do it with the other articles.
Or is this an exception?

Reply:
You are correct, I will update the capitalization.